inspectors/models/abstractinspectormodel.cpp

```diff
@@ -325,5 +331,8 @@ Ms::Score* AbstractInspectorModel::parentScore() const
         return nullptr;
     }
 
+    if (!m_elementList.at(0))
```

Comment:
если выбранных элементов нет, то в этом методе вообще нечего делать, для этого в самом начале есть
`if (!hasAcceptableElements())
        return;
        return;`

Reply:
This string checks whether the first element is not `nullptr` to avoid crash in next line. It has nothing in common with checking the list emptiness, but the first element correctness. This is more an assert anyway.